webapp/components/test-run.html

```diff
@@ -53,7 +58,15 @@
         <div>{{displayName(testRun.browser_name)}} {{shortVersion(testRun.browser_name, testRun.browser_version)}}</div>
         <template is="dom-if" if="{{ !isDiff(testRun.browser_name) }}">
           <div>{{displayName(testRun.os_name)}} {{testRun.os_version}}</div>
-          <div class="revision">@<a href="?sha={{testRun.revision}}">{{testRun.revision}}</a></div>
+          <template is="dom-if" if="[[githubCommitLinks]]">
+            <div class="github">
+              <img src="/static/github.svg" />
```

Comment:
Can you sprinkle some CSS so that this icon is vertically aligned with the text no matter the text size? Seems a few px off now.